tests/functional/object/rgw/test_obc_quota.py

```
@@ -121,6 +121,96 @@ def test_rgw_obc_quota(
+                    tier1,
+                    pytest.mark.polarion_id("OCS-XXXX"),
+                ],
+            ),
+        ],
+    )
+    def test_rgw_obc_size_quota(
+        self,
+        awscli_pod_session,
+        rgw_bucket_factory,
+        test_directory_setup,
+        mcg_obj_session,
```

Ruff (0.15.21)

[warning] Unused method argument: `mcg_obj_session`

(ARG002)